static/src/javascripts/projects/common/modules/commercial/creatives/fabric-expanding-v1.js

```diff
@@ -186,9 +186,11 @@ define([
             showPlus: (this.params.showMoreType === 'plus-only' || this.params.showMoreType === 'plus-and-arrow') ?
             '<button class="ad-exp__close-button ad-exp__open">' + svgs('closeCentralIcon') + '</button>' : ''
         };
+        var parallaxScrollType = this.params.backgroundImagePType;
+        var scrollbgDefaultY = '0%';
         var scrollingbg = {
-            scrollbg: (this.params.backgroundImagePType !== '' || this.params.backgroundImagePType !== 'none') ?
-            '<div class="ad-exp--expand-scrolling-bg" style="background-image: url(' + this.params.backgroundImageP + '); background-position: ' + this.params.backgroundImagePPosition + ' 50%; background-repeat: ' + this.params.backgroundImagePRepeat + ';"></div>' : ''
+            scrollbg: (parallaxScrollType !== '' || parallaxScrollType !== 'none') ?
```

Comment:
Should this not be an `&&`?
(I'm assuming that the empty string and 'none' are synonymous)